src/components/Button.js

```diff
@@ -51,7 +52,7 @@ const Button = props => {
     };
   }
   return (
-    <div style={styleButton}>{buttonName}</div>
+    <button style={styleButton} onClick={() => onClick(buttonName)} type="button">{buttonName}</button>
```

Comment:
I don't seem to know where `onClick` function came from, I can't see where you declared it.

Reply:
thanks